$OpenBSD: patch-src_bootstrap_bin_rustc_rs,v 1.9 2019/04/17 06:55:59 semarie Exp $
try to reduce memory usage on aarch64:
 - reduce memory use by retaining fewer names within compilation artifacts
Index: src/bootstrap/bin/rustc.rs
--- src/bootstrap/bin/rustc.rs.orig
+++ src/bootstrap/bin/rustc.rs
@@ -177,6 +177,11 @@ fn main() {
             cmd.arg("-C").arg(format!("codegen-units={}", s));
         }
 
+        if target.contains("aarch64-unknown-openbsd")
+            && crate_name == "rustc" {
+            cmd.arg("-Z").arg("fewer-names");
+        }
+
         // Emit save-analysis info.
         if env::var("RUSTC_SAVE_ANALYSIS") == Ok("api".to_string()) {
             cmd.arg("-Zsave-analysis");
